internal/build/build.go

```diff
@@ -0,0 +1,56 @@
+package build
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"runtime/debug"
+	"strings"
+	"time"
+
+	"github.com/cli/safeexec"
+)
+
+var (
+	// Version is dynamically set by the toolchain or overridden by the Makefile.
+	Version = "DEV"
+	// Date is dynamically set at build time in the Makefile.
+	// YYYY-MM-DD
+	Date = ""
+	// 2006-01-02T15:04:05Z07:00
+	Time = time.Now().UTC().Format(time.RFC3339)
+	// v2.10.1-19-ge837b7bc
+	Tag = ""
+)
+
+func init() {
+	if Version == "DEV" {
+		if info, ok := debug.ReadBuildInfo(); ok && info.Main.Version != "(devel)" {
+			Version = info.Main.Version
+		}
+	}
+
+	if tag, err := cmdOutput("git", "describe", "--tags"); err == nil {
```

Comment:
so this relies on git being installed and pulling this data from git. I wouldn't do that

Reply:
Resolved. Please see latest commit.

Please, mark as resolved, if you are happy with it.

Thanks.